webapp/angular.json

```diff
@@ -100,7 +101,8 @@
               "src/tsconfig.spec.json"
             ],
             "exclude": [
-              "**/node_modules/**"
+              "**/node_modules/**",
+              "**/*.json"
```

Comment:
As `build-angular:tslint` respects `tsconfig.json`, there is no need to exclude things (especially JSON ones)

Reply:
`src/lib/cothority/protobuf/models.json` gives errors in linting. Will disappear if I get around to replace `make src` with `npm link ../dynacred && inotify...`